docs/dev/todo.rst

```diff
@@ -42,6 +42,7 @@ Requests currently supports the following versions of Python:
 - Python 3.4
 - Python 3.5
 - PyPy 1.9
+- PyPy 2.2
```

Comment:
I think there is no value in being specific about PyPy versions: we just support PyPy.